src/components/Missions.js

```diff
@@ -1,7 +1,18 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useState } from 'react-redux';
+import { retrieveMissions } from '../Redux/missions/missions';
 
-const Missions = () => (
-  <div>THis is Mission section</div>
-);
+const Missions = () => {
+  const missions = useState((state) => state.missions);
+  const dispatch = useDispatch();
```

Comment:
Hello @patriciachrysy . Great job so far. I get the error in the image below when I run `npm start`. It is proabaly because you are importing `useState` from `react-redux` instead of `react`. Kindly go over it please.

![image](https://user-images.githubusercontent.com/94551484/181010621-0c64150c-7248-40e0-b1eb-5b98bf328518.png)

You can ask questions if anything is not clear. Thank you.

Reply:
I fixed it